raiden-dapp/src/services/raiden-service.ts

```diff
@@ -49,14 +50,21 @@ export default class RaidenService {
     stateBackup?: string,
     subkey?: true,
   ): Promise<Raiden> {
+    const { default: adapterPlugin } = await import('pouchdb-adapter-cordova-sqlite');
```

Comment:
@weilbith Do we already have a way to decide whether or not we're building for mobile?

Reply:
We have a dedicated environment for it. We could make the adapter a configuration. And then handle it conditionally. This would theoretically allow to even set other handlers later without effort.

Reply:
> We could make the adapter a configuration. And then handle it conditionally.

With that you mean enabling is based on some environment variable?

Reply:
I would not make it a boolean but rather just put the adapter name into a variable. If this variable is defined, do all the stuff you added with this variable as "generic". If not (default) just skip it and let the SDK do its default stuff.